app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -2,6 +2,7 @@ module ServiceOffering
   class AddToPortfolioItem
     include SourceMixin
     IGNORE_FIELDS = %w[id created_at updated_at portfolio_id tenant_id].freeze
+    ServiceOfferingIconFile = Struct.new(:tempfile)
```

Comment:
Is this a class declaration or a constant?

Reply:
I agree with @syncrou here, is there anything this is buying us over just using `Struct.new(:tempfile => file)` down on L65?

Reply:
Changed to use `OpenStruct` inline so we don't have to have the class declaration.